### ISSUE TITLE
TypeError: the JSON object must be str, not 'bytes'  in python 3.5 

### ISSUE THREAD

**Issue by nicocesar** [nsqio/pynsq#166]
Python 3.5 gives the following error 

```
[I 161217 20:27:01 reader:235] [test:test] starting reader for test/test...
[E 161217 20:27:01 ioloop:633] Exception in callback functools.partial(<function wrap.<locals>.null_wrapper at 0x7f021abac950>, HTTPResponse(_body=b'{"status_code":200,"status_txt":"OK","data":{"channels":["test"],"producers":[{"remote_address":"10.0.1.5:43360","hostname":"759d542da5ec","broadcast_address":"759d542da5ec","tcp_port":4150,"http_port":4151,"version":"0.3.8"}]}}',buffer=<_io.BytesIO object at 0x7f021ac1ceb8>,code=200,effective_url='http://nsq_lookup:4161/lookup?topic=test',error=None,headers=<tornado.httputil.HTTPHeaders object at 0x7f021abb2588>,reason='OK',request=<tornado.httpclient.HTTPRequest object at 0x7f021abb2320>,request_time=0.013861417770385742,time_info={}))
    Traceback (most recent call last):
      File "/usr/lib/python3.5/site-packages/tornado/ioloop.py", line 604, in _run_callback
        ret = callback()
      File "/usr/lib/python3.5/site-packages/tornado/stack_context.py", line 275, in null_wrapper
        return fn(*args, **kwargs)
      File "/usr/lib/python3.5/site-packages/nsq/reader.py", line 597, in _finish_query_lookupd
        lookup_data = json.loads(response.body)
      File "/usr/lib/python3.5/json/__init__.py", line 312, in loads
        s.__class__.__name__))
    TypeError: the JSON object must be str, not 'bytes'

```

this is an adaptation from nsq_to_nsq.py that only reads and prints

```
import tornado.options
from nsq import Reader, run
from nsq import Writer, Error
import functools
import logging
from host_pool import HostPool


class NSQProxy:
    def __init__(self, topic):
        self.topic = topic

    def relay(self, nsq_message):
        nsq_message.enable_async()
        print(nsq_message.body)        

if __name__ == "__main__":
    tornado.options.define('topic', type=str)
    tornado.options.define('lookupd_http_address', type=str) ##, multiple=True) TODO add multiple support 
    tornado.options.define('lookupd_http_port', type=int, default=4161)
    tornado.options.define('channel', type=str)
    tornado.options.define('max_in_flight', type=int, default=500)

    tornado.options.parse_command_line()

    assert tornado.options.options.topic
    assert tornado.options.options.lookupd_http_address
    assert tornado.options.options.channel

    destination_topic = str(tornado.options.options.topic)
    lookupd_http_addresses = "http://%s:%s" % (str(tornado.options.options.lookupd_http_address),
                                               str(tornado.options.options.lookupd_http_port))

    proxy = NSQProxy(destination_topic)

    Reader(
        topic=tornado.options.options.topic,
        channel=tornado.options.options.channel,
        message_handler=proxy.relay,
        max_in_flight=tornado.options.options.max_in_flight,
        lookupd_http_addresses=lookupd_http_addresses
    )
    run()
```

This is the Docker file 
```
FROM gliderlabs/alpine

RUN apk add --no-cache python3
RUN apk add --no-cache py-pip
RUN pip3 install --upgrade pip

## For debugging puporses
RUN pip3 install ipdb

RUN mkdir /opt
ADD requirements.txt /opt/requirements.txt
RUN cd /opt && pip3 install -r requirements.txt
```

and requirements.txt
```
argparse==1.2.1
tornado==4.4.2
pynsq==0.7.0
host_pool==0.2
```

**Comment by ploxiln** [comment]
fixed in #170 which was merged after the 0.7.0 release - unfortunately there hasn't been a release since then

**Comment by mreiferson** [comment]
just released v0.7.1!
